packages/-build-infra/src/cli-flags.js

```diff
@@ -1,27 +1,22 @@
 'use strict';
 
 function isInstrumentedBuild() {
-  return process.argv.includes('--instrument');
+  return process.env.INSTRUMENT;
 }
 
 function wantsEnabledFeatures() {
-  return process.argv.includes('--enable-in-progress');
+  return process.env.ENABLE_IN_PROGRESS;
 }
 
 function getManuallyEnabledFeatures() {
-  let args = process.argv;
   let enabled = {};
-  let ARG = '--enable-in-progress-flag';
+  let ARGS = process.env.ENABLE_IN_PROGRESS;
 
-  for (let i = 0; i < args.length; i++) {
-    if (args[i].indexOf(ARG) === 0) {
-      let toEnable = args[i].substr(ARG.length + 1).split(',');
-      toEnable.forEach(function(flag) {
-        enabled[flag] = true;
-      });
-      break;
+  ARGS.split(',').forEach(function(flag) {
```

Comment:
since the environment variable may not be present, we need to guard this behind a check that we actually have a value here